packages/teleport/src/stores/storeUserContext.ts

```diff
@@ -130,6 +130,10 @@ export default class StoreUserContext extends Store<UserContext> {
     return tokens.create;
   }
 
+  getAccessDownloadCenter() {
+    return this.state.acl.license.list || this.state.acl.download.list;
```

Comment:
i hunted this down:

```
- Users are able to download binaries and licenses on the web UI
- Downloading binaries is only allowed for users with permission to read and list the resource download in their roles
- Downloading licenses is only allowed for users with permission to read and list the resource license in their roles
```

couple of things:
- the drop down and selector is not centered (image below)
- should user have both `read` and `list` perms?
- first impression of the design (below image), it's pretty confusing, i think it should be like two separate sections `Download License` (and description) and `Download Binaries` especially if we want to handle the below edge case:

i'm probably pointing out a rare edge case where it will rarely be that a user will have `download` perm but not the `license` perm and vice versa, but do we want to handle the case?

![image](https://user-images.githubusercontent.com/43280172/209906671-68e46c83-fd17-4dfb-a479-5a2f8b794b72.png)

Reply:
this also confused me too, the `failure to download license` error (b/c this error seems to belong to the table): 
![image](https://user-images.githubusercontent.com/43280172/209907628-580914c9-39ce-49ff-a49f-9306b202fcce.png)

Reply:
maybe something like this perhaps:
![image](https://user-images.githubusercontent.com/43280172/209908324-85439221-e3cd-4faf-ba23-75206049e1c1.png)

Reply:
> the drop down and tab selector is not centered, and the tab icons for OS is also not centered (image below)

Fixed, thanks

> should user have both read and list perms?

I will double-check this with Xin. I think `list` should be all you need to download, while `read` for license (since there is only one license, nothing to list).


> first impression of the design (below image), it's pretty confusing, i think it should be like two separate sections Download License (and description) and Download Binaries especially if we want to handle the below edge case:

I have a sync with @roraback today, will bring this up and show your feedback. I agree that the error state will be confusing.

Reply:
Thanks for this great feedback @kimlisa ! Some thoughts:
* Support: tldr; I think we should hide the support page from this Houston replacement, because the links are mostly geared toward Teleport _users,_ whereas anyone in this UI is likely focused on deploying their self-hosted cluster. I'm open to other opinions, though, if folks disagree!
  * Original thought behind this was that with fewer options in the nav, it would make sense to include a support link there for greater visibility
  * I think I asked in an early discussion whether a help page would be useful, and the input was along the lines that a help page isn't crucial at this point for getting people off Houston (since they don't currently have a help page in Houston), and that the existing one in Teleport isn't so well suited to helping with the use case at hand.
  * All that said, I *am* somewhat conflicted on this. Part of me would love to reinforce the existing Teleport nav structure around help rather than create a new one. I think longer-term, it could be helpful to mirror the existing help page with something geared specifically toward enterprise deployment (and license update) needs.
* Binaries + License Key Permissions Errors: Agreed that this could confuse users. I'm going to come up with some gentler designs + messaging to inform users if they don't have permissions to access one or both download types.
* Separate tabbed structure for Download types: that feels like overkill to me, since there would just be two tabs and one of them wouldn't need to hold much more than a single button; and users often overlook secondary/tertiary tabs, so this could have the effect of burying the content in whichever tab isn't displayed by default.
  * That said, perhaps a bit of guidance around what to do with the license key—which Houston does NOT do currently—could call it out a bit more clearly as a separate piece from the binaries? My understanding is that our enterprise customers are typically walked through these deployment steps by Teleport during the POV phase, so I'm a bit unsure how much value it adds, but it would be low effort to include and might at least avert some confusion around updating the license.

I tried to respond to everything above, but let me know if I missed anything or if you have additional questions or concerns.

Reply:
Here is the design I came up with for clarifying the role of the license key and binary downloads: 
![image](https://user-images.githubusercontent.com/1095531/210416206-422969b2-5c5f-4de0-b034-e1d191741d1e.png)